www/components/shelf/shelf.js

```diff
@@ -22,29 +22,24 @@ class Shelf extends LitElement {
     this.page = '';
   }
 
-  connectedCallback() {
+  async connectedCallback() {
     super.connectedCallback();
-    this.collapseAll();
-    this.expandRoute(window.location.pathname);
-  }
-
-  async setupShelf(page) {
-    console.log('setupShelf for page =>', page);
-
-    if (page && page !== '' && page !== '/') {
-      // TODO remove require call - #275
-      this.shelfList = require(`./${page}.json`);
-
-      // TODO not actually integrated, still using .json files - #271
```

Comment:
This line is what I'm referring to when I resolved 271, using my method.  You knew we were going to use a query here.